src/game/boe.graphics.cpp

```diff
@@ -1083,7 +1080,7 @@ static void init_trim_mask(std::unique_ptr<sf::Texture>& mask, rectangle src_rec
 //short which_mode;  // 0 top 1 bottom 2 left 3 right 4 up left 5 up right 6 down right 7 down left
 void draw_trim(short q,short r,short which_trim,ter_num_t ground_ter) {
 	/* TODO: Windows has a check for frills being enabled:
-	if(!frills_on || (((current_ground == 2) || (current_ground == 36)) && (PSD[SDF_COMPATIBILITY_FULL_TRIMS] == 0)))
+	if(((current_ground == 2) || (current_ground == 36)) && (PSD[SDF_COMPATIBILITY_FULL_TRIMS] == 0))
```

Comment:
Let's remove the entire TODO note here.

* The PSD at the end is sort of a duplicate of the preference being removed (or might be a slightly related preference that was unique to the Windows version)
* I don't really understand the checks for current_ground, but they seem wrong – hardcoding no frills on two terrains can't be right, just set those terrains to not have frills.